### PR TITLE
update albion online

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/71333)

### CODE REVIEW

**Comment by solomoncyj** [comment]
har?

**Comment by mdanish-kh** [comment]
Try changing the architecture to `x86` (Task Manager shows the app's architecture is x86). Manifest schema version has nothing to do with this error, so you can revert back to `1.2.0`.

**Comment by solomoncyj** [comment]
.

File: manifests/a/Albion/Online/22.8.9/Albion.Online.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: Albion.Online
+PackageVersion: 22.8.9
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+ReleaseDate: 2022-08-09
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/a/Albion/Online/22.8.9/Albion.Online.installer.yaml:12]:
```suggestion
- Architecture: x64
```

File: manifests/a/Albion/Online/22.8.9/Albion.Online.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS2.7-3-0-preview-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Albion.Online
+PackageVersion: 22.8.9
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+ReleaseDate: 2022-08-09
```
Comment (ItzLevvie) [inline manifests/a/Albion/Online/22.8.9/Albion.Online.installer.yaml:10]:
```suggestion
ReleaseDate: 2022-08-09
InstallerSuccessCodes:
- 1223
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/Albion/Online/22.8.9/Albion.Online.installer.yaml (hunk 1 above)
Comment (solomoncyj) [reply manifests/a/Albion/Online/22.8.9/Albion.Online.installer.yaml]: https://github.com/microsoft/winget-pkgs/pull/71333#issuecomment-1221335094

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**